frontend/src/views/CanteensPage/PublishedCanteenCard.vue

```diff
@@ -10,22 +10,26 @@
     <v-card-subtitle v-if="canteen.dailyMealCount || canteen.city">
       <CanteenIndicators :canteen="canteen" />
     </v-card-subtitle>
-    <v-card-text
-      v-if="initiativesForCanteen(canteen) && initiativesForCanteen(canteen).length > 0"
-      class="grey--text text--darken-4"
-    >
-      Nos initiatives mises en place :
-      <div v-for="initiative in initiativesForCanteen(canteen)" :key="initiative">
-        <v-icon small class="mt-n1" color="primary">mdi-check-bold</v-icon>
-        {{ initiative }}
-      </div>
+    <v-card-text v-if="Object.keys(earnedBadges).length" class="grey--text text--darken-4 mx-1 mt-2">
+      <v-row>
+        <v-img
+          max-width="30"
```

Comment:
```suggestion
          max-width="25"
```
Un chouya plus petit pour que ça prenne un peu moins d'espace vertical (vu que les cartes adjacentes sont en "fill-height"). T'en penses quoi ?

D'ailleurs super fan de cette UI <3